schemas/observation/digital-signature.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var Joi = require( 'joi' );
+
+var signatureInfo = {
+	'sign'    : Joi.string().optional().description( 'string that the user inputs for digital signatures' ),
+	'date'    : Joi.date().optional().description( 'The date of which the user signed the observation' ),
+	'isValid' : Joi.boolean().optional().description( 'flag that determines if the signature is valid or not' )
+};
+
+module.exports = {
+	'observer' : Joi
+		.array()
+		.optional()
+		.items( Joi
+			.object()
+			.optional()
+			.keys( signatureInfo ) )
+		.description( 'digital signature info for observer' ),
+
+	'observee' : Joi
```

Comment:
Just to confirm: the only **two** people that will be digitally-signing an observation are the observer and the observee, correct?